### PR TITLE
More floors and decals

### PR DESCRIPTION
**Pull request by DreamySkrell**
image

changes:
  - rscadd: "Adds floors, and recolores of existing floors."
  - rscadd: "Adds floor markings and decals."

### CODE REVIEW

**Comment by DreamySkrell** [comment]
!wip

File: code/game/turfs/flooring/flooring_premade_.dm [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+	// initial_flooring = /singleton/flooring/tiling/dark
+	tile_decal_state = "techmaint_light"
+	color = COLOR_GRAY30
+
+/turf/simulated/floor/tiled/techfloor
+	name = "techfloor tiles"
+	icon_state = "techfloor"
+	// initial_flooring = /singleton/flooring/tiling/dark
+	tile_decal_state = "techfloor_light"
+	color = COLOR_GRAY40
```
Comment (Arrow768) [inline code/game/turfs/flooring/flooring_premade_.dm:300]: Not defined, apparently

**Review by Arrow768 — CHANGES_REQUESTED** [review]

**Comment by DreamySkrell** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]
I hope these have proper construction/crafting recipes

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**